nomt/src/beatree/ops/mod.rs

```diff
@@ -21,21 +21,37 @@ mod update;
 pub use reconstruction::reconstruct;
 pub use update::update;
 
+/// Partially look up a key in the btree. This will determine the leaf node page the leaf is within.
```

Comment:
It does what it says on the tin - it determines the leaf node page the leaf is within.

This is not public API

Reply:
Like I said: I think I do understand what it does, but definitely not after reading from the tin. My questions are meant to highlight that 

a) IMO the comment and the naming is sub par
b) meant to help to guide to a better solution

In general, in my value system, good comment >>> no comment >>> bad comment. IOW if a comment slows you down and drives you astray it would be better to just let the reader read the code.

Reply:
(I fixed it anyway)